src/screens/Home/TaskItem.js

```diff
@@ -1,49 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { StyleSheet, View, Text } from 'react-native';
-import { Button } from 'react-native-elements';
 import Colors from '../../styles/colors';
+import SurveyQuestion from './SurveyQuestion';
 
-const StyledButton = ({title}) =>
-  <Button
-    raised
-    borderRadius={2}
-    title={title}
-    containerViewStyle={{ flex: 1, width:'50%'}}
-    buttonStyle={{ backgroundColor: Colors.red, height: 36 }}
-    titleStyle={{color:Colors.white, fontSize: 14}}
-  />
-
-const TaskItem = ({heading, content}) =>
-  <View style={styles.taskItem}>
-    <Text style={styles.heading}>{heading}</Text>
-    <Text style={styles.content}>{content}</Text>
-    <View style={styles.buttonBox}>
-      <StyledButton title="Yes"/>
-      <StyledButton title="No"/>
-    </View>
-  </View>
+class TaskItem extends Component {
+  render(){
+    const { heading, content, updateResponse, userResponse } = this.props;
+    return(
+      <View style={styles.taskItem}>
+        {
+          !userResponse ?
+          <SurveyQuestion {...this.props}/>
+          :
+          <View>
+            <Text>63% of Californians Agree!</Text>
```

Comment:
I would probably recommend putting as much of what will eventually dynamic as constants at the top of the file (and setting to props) to minimize the amount of work to switch over to dynamic data.  It will make it easier to see what we need to make sure we have in database/selectors/understand the shape it needs to be in, and also eliminate a lot of busy work tracking down all the hardcoded values and setting them to props

Reply:
Thanks for the notes! I'll take a stab at selectors for these and use your candidate detail changes as a guide.